src/components/App.tsx

```diff
@@ -2885,8 +2885,15 @@ class App extends React.Component<ExcalidrawProps, AppState> {
           mutateElement(draggingElement, { points: [...points, [dx, dy]] });
         } else if (points.length > 1) {
           if (draggingElement.type === "draw") {
+            /**
+             *  the constant .1 is fairly arbitrary but was chosen to keep
+             *  a max distance of 1 (.1 is the farthest zoom out).
+             **/
             mutateElement(draggingElement, {
-              points: simplify([...(points as Point[]), [dx, dy]], 0.7),
+              points: simplify(
+                [...(points as Point[]), [dx, dy]],
+                0.1 / this.state.zoom,
```

Comment:
Can we change this value so that at zoom 1 the granularity stays the same as before?